packages/serial/src/message/Constants.ts

```diff
@@ -52,8 +52,8 @@ export enum FunctionType {
 	UNKNOWN_FUNC_MEMORY_PUT_BUFFER = 0x24,
 
 	EnterBootloader = 0x27, // Leave Serial API and enter bootloader (700+ series only). Enter Auto-Programming mode (500 series only).
-	UNKNOWN_FUNC_UNKNOWN_0x28 = 0x28, // ??
-
+	UNKNOWN_FUNC_UNKNOWN_0x28 = 0x28, // ZW_NVRGetValue(offset, length) => NVRdata[], see INS13954-13
+ 
```

Comment:
Unrelated to long range, but I found a reference doc from Silicon Labs 500 series chips that gave the `FunctionType` values for a lot of unknown ones.